No need to run-depend on setuptools_rust and rust..

Index: pyproject.toml
--- pyproject.toml.orig
+++ pyproject.toml
@@ -217,7 +217,7 @@ pydantic = ">=1.7.4"
 # This isn't really a dev-dependency, as `poetry install --no-dev` will fail,
 # but the alternative is to add it to the main list of deps where it isn't
 # needed.
-setuptools_rust = ">=1.3"
+# setuptools_rust = ">=1.3"
 
 
 # Optional Dependencies
